### PR TITLE
Provide a script to populate the registry instance

### PR DESCRIPTION
**Pull request by candleindark**
Provide a script to populate a Datalad registry instance with the Datalad datasets listed on [Datalad usage dashboard](https://github.com/datalad/datalad-usage-dashboard/).

Currently, the script can only populate a Datalad registry instance with active datasets on GitHub listed on [Datalad usage dashboard](https://github.com/datalad/datalad-usage-dashboard/). However, it can be easily extended for population with datasets on OSF listed on  [Datalad usage dashboard](https://github.com/datalad/datalad-usage-dashboard/) in the future.

### CODE REVIEW

**Comment by candleindark** [comment]
To get this PR to pass the CI, it needs to change from https://github.com/datalad/datalad-registry/pull/91. I will rebase this branch on master once https://github.com/datalad/datalad-registry/pull/91 is merged to master so that the commit history look more relevant.

File: tools/populate.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+# Build clone URLs for active GitHub datasets
+active_github_dataset_urls = [ds.url + ".git" for ds in active_github_datasets]
+
+# Submit URLs of active datasets to the datalad-registry
+start: Optional[int] = None  # Modify this to adjust the list of URLs to submit
+stop: Optional[int] = None  # Modify this to adjust the list of URLs to submit
+s = slice(start, stop)
+registry_submit_urls = RegistrySubmitURLs()
+registry_submit_urls(active_github_dataset_urls[s])
```
Comment (yarikoptic) [inline tools/populate.py:91]: so how did this work out?

Comment (candleindark) [reply tools/populate.py:91]: Unfortunately not good. Please see https://github.com/datalad/datalad-registry/issues/93 for details.

**Comment by candleindark** [comment]
I have rebased the branch, and this PR is now ready for merge.

**Comment by yarikoptic** [comment]
> @yarikoptic I have rebased the branch, and this PR is now ready for merge.

please make it into a "proper" script  with `click` interfacing options for choice of the range, and `if __name__ ...` check (so if just imported -- doesn't run) . See e.g. any of the tools within dandisets, e.g. https://github.com/dandi/dandisets/blob/78c7267fc942914164fe1949bf9c2122753aa3a4/tools/get-upload-stats.py, for an example

**Comment by yarikoptic** [comment]
ping on the comment above

File: datalad_registry/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+from .misc import StrEnum  # noqa: F401(imported but unused)
```
Comment (yarikoptic) [inline datalad_registry/utils/__init__.py:1]: not clear why bother importing here then at all

Comment (candleindark) [reply datalad_registry/utils/__init__.py:1]: I was thinking making `StrEnum` immediately available under the `utils` package.

**Pull request merged**